mdesnoyer/wonderland
# Email Results Wireup #1353 #1386 #1393

# Changes
- added email functionality to `ShareEmail`
- moved `ShareLink` functionality into `VideoContent` as it was also need by `ShareEmail`
- hide `VideoCollectionActions` from guest users
# Test Plan
- sign in to a normal account
  - click the share email icon for any video and enter a valid email
  - check that the form is replaced by a success message and the "back" button is still there
  - check that you receive and email with all the links functioning and images appearing.
- go to a shared video link
  - check that the share links are not visible
